app/design/adminhtml/Magento/backend/Magento_Bundle/web/css/source/_module.less

```diff
@@ -0,0 +1,15 @@
+// /**
+//  * Copyright © Magento, Inc. All rights reserved.
+//  * See COPYING.txt for license details.
+//  */
+
+
+.admin__dynamic-rows {    
```

Comment:
This selector is too general and can impact element that are not related to bundle product, can you please make it more specific